kola/tests/util/containers.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/platform"
+)
+
```

Comment:
Could some comments be added here about what this is doing or why you would use it?

Reply:
Would `GenPodmanScratchContainer` convey the intent of the function better?

Reply:
I changed the name of the function and also added a comment for the exported function (golint told me I should have done that).